airtable.js

```diff
@@ -0,0 +1,45 @@
+/* global useRuntimeConfig, $fetch */
+
+const services = []
+
+const setRecords = (records) => {
+  records.forEach((element) => {
+    services.push(element)
+  })
+}
+
+export const getServices = async () => {
+  const config = useRuntimeConfig()
+  const options = {
+    headers: {
+      Authorization: `Bearer ${config.API_KEY}`,
+    },
+  }
+  let offset = null
+  do {
+    const offSet = offset ? `offset=${offset}` : ''
```

Comment:
What is offset doing?

Reply:
that's a parameter of the api, used for pagination